pkg/remote/github/provider.go

```diff
@@ -0,0 +1,60 @@
+package github
+
+import (
+	"os"
+
+	"github.com/cloudskiff/driftctl/pkg/remote/terraform"
+	tf "github.com/cloudskiff/driftctl/pkg/terraform"
+)
+
+type GithubTerraformProvider struct {
+	*terraform.TerraformProvider
+}
+
+type githubConfig struct {
+	Token        string
+	Owner        string `cty:"owner"`
+	Organization string
+}
+
+func NewGithubTerraformProvider() (*GithubTerraformProvider, error) {
+	p := &GithubTerraformProvider{}
+	providerKey := "github"
+	installer, err := tf.NewProviderInstaller(tf.ProviderConfig{
+		Key:     providerKey,
+		Version: "4.4.0",
+	})
+	if err != nil {
+		return nil, err
+	}
+	tfProvider, err := terraform.NewTerraformProvider(installer, terraform.TerraformProviderConfig{
+		Name:         providerKey,
+		DefaultAlias: p.getDefaultOwner(),
+		GetProviderConfig: func(owner string) interface{} {
```

Comment:
Shouldn't it be `func(alias string) interface{} {`

Reply:
I've put owner there to make this parameter maybe more clear in github context.

* AWS alias = region
* Github alias = owner

I can totally put alias everywhere if you find it more clear

Reply:
I'm fine with that